### PR TITLE
Undo button

### LINKED ISSUE #36: Undo button
From https://github.com/umts/transit-it/issues/677.

It'd be nice if changes to single events could be rolled back.

http://railscasts.com/episodes/255-undo-with-paper-trail might be a good guide, though the actual code is quite out of date.

### PR DESCRIPTION
**Pull request by dfaulken**
Closes #36.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 language: ruby
-rvm: 2.3.0
+rvm: 2.3.1
 env: 
-  - 'CODECLIMATE_REPO_TOKEN=029db8a4d83b74b2b01b09670c9a28a9c353175415b61186e87a1d70fe461931'
-  - 'RAILS_ENV=test'
+  - 'CODECLIMATE_REPO_TOKEN=029db8a4d83b74b2b01b09670c9a28a9c353175415b61186e87a1d70fe461931 RAILS_ENV=test'
```
Comment (Anbranin) [inline .travis.yml]: ?

Comment (dfaulken) [inline .travis.yml]: It was running two builds, one for each `env` setting. This runs them together.

File: app/controllers/changes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ChangesController < ApplicationController
+  def undo
+    version = PaperTrail::Version.find(params.require :id)
+    head :unauthorized and return unless version.done_by? @current_user
+    # Reify only returns false when the thing didn't exist beforehand.
+    if version.reify
+      version.reify.save!
```
Comment (Anbranin) [inline app/controllers/changes_controller.rb]: Cool. People can't make changes to multiple objects at once with one write, can they?

Comment (dfaulken) [inline app/controllers/changes_controller.rb]: They can (see the undo comment I put on generating rotations), but those would be multiple versions anyway.

**Pull request merged**